<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configFileOption</class>
 <widget class="QDialog" name="configFileOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Config File Path</string>
  </property>
  <widget class="QPushButton" name="pbSet">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>90</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenPath">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>31</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Select A Folder To Create Config File In(Optional).</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pbOpenPath</tabstop>
  <tabstop>pbSet</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
